layout/templates/walleyeDecline.mustache

```diff
@@ -8,14 +8,10 @@
 <div id="walleyeDecline">
 		
 		<ul class='list'>
-		  <li>{{ walleyeDecline-l1 }}</li>
-		  <li>{{ walleyeDecline-l2 }}</li>
-		  <li>{{ walleyeDecline-l3 }}</li>
-		  <li>{{ walleyeDecline-l4 }}</li>
-		  <li>{{ walleyeDecline-l5 }}</li>
-		</ul>
+	  {{#walleyeDecline-bullets}}  <li>{{.}}</li>
+	  {{/walleyeDecline-bullets}}</ul>
 		
-	  <object id="walleyeDeclineSVG" type="image/svg+xml" class="svgFig" data="{{ wallyTrendsFig }}">
```

Comment:
remove class so this stretches to full width. Probably will want a different class for this one in the future